demo/src/app/app.component.ts

```diff
@@ -117,14 +117,19 @@ export class AppComponent implements OnInit {
     charCounterCount: false,
     toolbarInline: true,
     events: {
-      'froalaEditor.initialized': function() {
-        console.log('initialized');
-      },
-      "froalaEditor.contentChanged": (e: any, editor: any) => {
-        console.log(editor)
-      }
-    }
+      'froalaEditor.initialized': function goToEnd(e, editor) {
+        if ($('#froala-editor').froalaEditor('core.isEmpty')) {
+          $('#froala-editor').froalaEditor('events.focus');
+        }
+        else{
+          editor.events.on('click', function() {
```

Comment:
Placing the cursor at the end of the editor on click is not a solution. This would prevent placing the cursor in middle of text.